mod/aalborg_theme/views/default/page/elements/navbar.php

```diff
@@ -9,7 +9,7 @@
 
 ?>
 
-<a class="elgg-button-nav" rel="toggle" href=".elgg-nav-collapse">
+<a class="elgg-button-nav" rel="toggle" data-toggle-selector=".elgg-nav-collapse" href="#">
```

Comment:
This would ideally be a button element, but in a bugfix release I'm not inclined to change it. cc @PerJensen @ewinslow